tools/toolchains/__init__.py

```diff
@@ -742,12 +742,22 @@ def add_regions(self):
         except ConfigException:
             pass
 
+    def add_linker_defines(self):
+        stack_param = "target.boot-stack-size"
+        params, _ = self.config_data
+
+        if stack_param in params:
+            define_string = self.make_ld_define("MBED_BOOT_STACK_SIZE", int(params[stack_param].value, 0))
+            self.ld.append(define_string)
+            self.flags["ld"].append(define_string)
```

Comment:
@theotherjimmy this is how the target define is getting passed to the linker script. Is this the right way to do it or would you like any changes made here?

Reply:
That looks fine.